scripts/components/Interactions/HotspotNavButton.js

```diff
@@ -79,7 +79,7 @@ export default class HotspotNavButton extends React.Component {
     we have to multiply the result by two*/
 
     let finalValue = ((currentPosMouse - divStartWidth) * 2);
-    if(finalValue > 128 && finalValue < 512) {
+    if(finalValue > 128 && finalValue < 2048) {
```

Comment:
Will this affect height as well?

Reply:
yes

Reply:
divStartWidth may not be the correct word though

Reply:
Changed it to divStartPos